packages/block-editor/src/components/colors-gradients/style.scss

```diff
@@ -120,3 +121,22 @@ $swatch-gap: 12px;
 		flex-shrink: 0;
 	}
 }
+
+.block-editor-panel-color-gradient-settings__reset {
+	position: absolute;
+	display: inline-flex;
+	right: 0;
+	top: 0;
+	bottom: 0;
+	opacity: 0;
```

Comment:
Can we confirm that it's better if the button is hidden by default? 

I'm having second thoughts if this is good for discovery. A user might be wondering how to reset the color and wouldn't know unless they hover on the tools panel item.

Reply:
The intention was to [show the reset button only on hover or focus](https://github.com/WordPress/gutenberg/issues/41866#issuecomment-1203608605), making it easier to reset colors with fewer clicks.

From my perspective, showing the button on hover is a subtle and unobtrusive way to achieve this without adding visual clutter to the UI. As for discoverability, I think users will naturally figure it out when they try resetting colors the usual way, which should be sufficient.